components/hab/src/command/studio/docker.rs

```diff
@@ -129,9 +131,13 @@ fn find_docker_cmd() -> Result<PathBuf> {
 
 fn is_image_present(docker_cmd: &Path) -> bool {
     let mut cmd = Command::new(docker_cmd);
-    cmd.arg("images")
-        .arg(&image_identifier(docker_cmd))
-        .arg("-q");
+    let using_windows_containers = is_serving_windows_containers(&docker_cmd);
+    let image = image_identifier(
+        using_windows_containers,
+        target::PackageTarget::active_target(),
+    );
```

Comment:
It looks like this same block gets repeated several times, I wonder if it could be abstracted into a function.

Reply:
```
fn image_identifier_for_active_target(docker_cmd: &Path) -> String {
    image_identifier(
        is_serving_windows_containers(docker_cmd),
        target::PackageTarget::active_target(),
    )
}
```
This is what I'm thinking about, which seems odd but that may just be lack of experience.  That behavior was extracted out of `image_identifier` to make it testable, otherwise you are relying on OS state (windows containers) and how it was compiled (ActiveTarget) to get an image name.  This function would have the same testing issue, but at least the logic is encapsulated in a testable function now. 

Does that seem like the right change to make?